hydra-tui/src/Hydra/TUI.hs

```diff
@@ -682,10 +703,11 @@ style _ =
 runWithVty :: IO Vty -> Options -> IO State
 runWithVty buildVty options@Options{hydraNodeHost, cardanoNetworkId, cardanoNodeSocket} = do
   eventChan <- newBChan 10
```

Comment:
Suggestion: Use `MVar` instead of `BChan`, I don't see why we would want to have more than one `Tick` in queue. If the consumer is too slow, we should just stop producing ticks?

Reply:
Perhaps, instead of storing the remaining time, we could store the actual deadline and do the reverse computation in the display? This could lead to less surprising behavior to users in case there is a lag

Reply:
Perhaps a small step would be to store the `deadline` by just computing it in the TUI, using the time provided by `Tick` and then a difference with `contestationPeriod`?